packages/vincent-tools/vincent-tool-uniswap-swap/src/lib/tool-checks/check-erc20-allowance.ts

```diff
@@ -0,0 +1,35 @@
+import { PublicClient } from 'viem';
```

Comment:
🤔 hm. Kind of awkward to have this in both the erc20 allowance tool and this tool. I guess it's NBD